src/js/components/inspectors/Area.tsx

```diff
@@ -1,39 +1,68 @@
 'use strict';
 const INTERPOLATE = require('../../constants/interpolate');
+const getInVis = require('../../util/immutable-utils').getInVis;
 
 import * as React from 'react';
 import {connect} from 'react-redux';
 import {PrimType} from '../../constants/primTypes';
 import {Property} from './Property';
+import {ExtentProperty} from './ExtentProperty';
+import {State} from '../../store';
 
-interface AreaProps {
+const Area = {
+  'ORIENT': [
+    'vertical',
+    'horizontal'
+]};
+
+interface OwnProps {
   primId: number,
   primType: PrimType
 }
 
-class BaseArea extends React.Component<AreaProps> {
+interface StateProps {
+  orient: string;
+}
+
+function mapStateToProps(reduxState: State, ownProps: OwnProps): StateProps {
+  const primId = ownProps.primId;
+  return {
+    orient: getInVis(reduxState, `signals.lyra_area_${primId}_orient.value`)
+  };
+}
+class BaseArea extends React.Component<OwnProps & StateProps> {
   public render() {
     const props = this.props;
     return (
       <div>
-        {/* <div className="property-group">
+        <div className="property-group">
           <h3>Orientation</h3>
 
           <Property name="orient" label="Orient" type="select"
             opts={Area.ORIENT} {...props} />
-        </div>*/}
+        </div>
 
-        <Property name='x' type='number' droppable={true} {...props}>
-          <h3 className='label'>X Position</h3>
-        </Property>
+        {props.orient === 'vertical' ?
+            <Property name='x' type='number' droppable={true} {...props}>
+              <h3 className='label'>X Position</h3>
+            </Property>
 
-        <div className='property-group'>
-          <h3>Y Position</h3>
+          : <div className='property-group'>
+              <h3>X Position</h3>
+              <ExtentProperty exType='x' {...props} />
+            </div>
+        }
 
-          <Property name='y' label='Start' type='number' droppable={true} {...props} />
+        {props.orient === 'vertical' ?
+            <div className='property-group'>
+              <h3>Y Position</h3>
+              <ExtentProperty exType='y' {...props} />
 
-          <Property name='y2' label='End' type='number' droppable={true} {...props} />
-        </div>
+            </div>
+          :  <Property name='y' type='number' droppable={true} {...props}>
+                <h3 className='label'>Y Position</h3>
+              </Property>
+        }
```

Comment:
a formatting suggestion for this part, to get the divider lines to show up in the right places (they're attached to the `property-group` class)

```
        <div className='property-group'>
          <h3>X Position</h3>
          {props.orient === 'vertical' ?
              <Property name='x' label='X' type='number' droppable={true} {...props} />
            :
              <ExtentProperty exType='x' {...props} />
          }
        </div>

        <div className='property-group'>
          <h3>Y Position</h3>
          {props.orient === 'vertical' ?
              <ExtentProperty exType='y' {...props} />
            :
              <Property name='y' label='Y' type='number' droppable={true} {...props} />
          }
        </div>
```